app/src/androidTest/java/org/dhis2/usescases/filters/FilterTest.kt

```diff
@@ -146,12 +145,13 @@ class FilterTest : BaseTest() {
         eventWithoutRegistrationRobot(composeTestRule) {
             clickOnEventAtPosition(0)
         }
-        formRobot(composeTestRule) {
+        // Commented because FormRobot class wah eliminated as FormTest was also eliminated
+        /*formRobot(composeTestRule) {
```

Comment:
if this is no longer needed maybe we can remove it?

Reply:
I don't think so, because this test is going to be looked at at some point. So it's better to leave it as it is so when someone does look into this, knows how the tests really is coded.